### ISSUE TITLE
TopologyControlBar references missing sr-only class

### ISSUE THREAD

**Issue by mabashian** [patternfly/patternfly-react#6098]
See the code here: https://github.com/patternfly/patternfly-react/blob/main/packages/react-topology/src/components/TopologyControlBar/TopologyControlBar.tsx#L274

I don't believe this class is included anymore so the resulting button renders look like:

<img width="1669" alt="Screen Shot 2021-07-30 at 2 27 24 PM" src="https://user-images.githubusercontent.com/9889020/127706543-a281dce3-aaa6-4a2b-adf9-0675a356397b.png">

when an ariaLabel is included.  I believe the intention was for those spans to be hidden and only referenced by screen readers.

I'm running react-core 4.121.1 and react-topology 4.9.6.

**Comment by jessiehuff** [comment]
I agree, I don't think we use that class specifically. We use `class="pf-u-screen-reader"` so this should be updated. 👍
